components/card.css

```diff
@@ -0,0 +1,27 @@
+.quizapp__card {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  align-items: center;
+  border: solid 2px var(--color-scarlet);
+  border-radius: 10px;
+  background-color: var(--color-cream);
+  margin: 20px;
+  padding: 20px;
+}
+
+.quiz-question {
+  color: var(--color-scarlet);
+  text-align: center;
+}
+
+.answer-button {
+  display: flex;
+  border: solid 1px var(--color-gray);
+  background-color: var(--color-scarlet);
+  border-radius: 5px;
+  color: var(--color-cream);
+  margin-bottom: 5px;
+  width: 100px;
+  font-size: 18px;
+}
```

Comment:
Just a suggestion, you could add an extra css-file for the question and the answer button as well.

Reply:
I didn't fully understand that structure then. A *whole* file for each element?

Reply:
You could do that, I just checked the handout again (wasn't sure anymore) and the suggested structure looks like this :

[https://github.com/neuefische/bo-web-23-4/blob/main/sessions/css-structure/challenges-css-structure.md#separate-the-quiz-app-css-code-into-different-files](url)

Hope that helps ✌️

Reply:
the link doesn't work?